source/agora/node/Ledger.d

```diff
@@ -1982,3 +1990,43 @@ unittest
             mpv_stake.output.address).array.length == 0);
     }
 }
+
+// Coinbase only ConsensusData and blocks
```

Comment:
Please leave a little more detailed description of what the situation is and what the expectation is.